### ISSUE TITLE
Add Improving Minecraft

### ISSUE THREAD

**Issue by LuminaDragon** [kayila/minecraftmodpacks#11]
Please add improving minecraft for rotten flesh cooking and adamantine stuffs! :)

**Comment by kayila** [comment]
Usually I add rotten flesh -> leather furnace recipe through Minetweaker. Improving Minecraft doesn't add it. ;) (I'll add a ticket for it however because it is a nice thing to have) ( #12 )

Improving minecraft is what added in the wild cow/chicken/pigs/sheep... You _really_ want that?

**Comment by LuminaDragon** [comment]
yeah that's true, that part sucks.  Hmm I just like the adamantine because of the aspects for Thaumcraft.

**Comment by kayila** [comment]
Which aspect? We could potentially add/change those aspects on other stuff using MineTweaker too... Seems like you're asking to add a mod that changes a lot of things for the purposes of just adding one item that happens to have some thaumcraft aspect on it. :stuck_out_tongue_winking_eye:

**Comment by LuminaDragon** [comment]
It was mostly that it was more of just the instrumentus, meto, tutamen, and telum in the tools without the added tons of metallum and not as hard to make in bulk
